kola-denylist.yaml

```diff
@@ -12,3 +12,7 @@
    - s390x
 - pattern: coreos.ignition.journald-log
   tracker: https://github.com/coreos/coreos-assembler/issues/1173
+- pattern: multipath.day1
+  tracker: https://src.osci.redhat.com/rpms/coreos-installer/pull-request/28 
```

Comment:
Waiting for https://bugzilla.redhat.com/show_bug.cgi?id=1974411

Reply:
had to open https://github.com/openshift/os/pull/617 as i had already force pushed to that branch.